Kajabi/sage-lib
# Link - add rails component and updated docs

## Description

- [x] - updated doc props

## Test notes

No visual changes

### Steps for testing
1. Visit the rails link page: http://localhost:4000/pages/element/link.
2. Verify demo and docs
